packages/ruby/lib/readme/errors.rb

```diff
@@ -1,10 +1,12 @@
+# frozen-string-literal: true
```

Comment:
Having this here allows us to not have to append `.freeze` to every string literal.

Reply:
Does this mean string literals are automatically frozen for us, or does it mean rubocop errors are ignored for this file?

Reply:
Means that they're implicitly frozen for us with this comment.